<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>692</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="VTKRender2DWidget" name="widget_axial" native="true"/>
       </item>
       <item row="0" column="1">
        <widget class="VTKRender2DWidget" name="widget_coronal" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="VTKRender2DWidget" name="widget_sagittal" native="true"/>
       </item>
       <item row="1" column="1">
        <widget class="VTKRender3DWidget" name="widget_3d" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_openFile">
        <property name="text">
         <string>openFile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_reset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_imageProcess">
        <property name="text">
         <string>ImageProcess</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_blendMode">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>COMPOSITE_BLEND</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MAXIMUM_INTENSITY_BLEND</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MINIMUM_INTENSITY_BLEND</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AVERAGE_INTENSITY_BLEND</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ADDITIVE_BLEND</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ISOSURFACE_BLEND</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SLICE_BLEND</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VTKRender2DWidget</class>
   <extends>QWidget</extends>
   <header>vtkrender2dwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VTKRender3DWidget</class>
   <extends>QWidget</extends>
   <header>vtkrender3dwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
